contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5094,72 +5130,65 @@ static List *transformSelectIntoStmt(CreateTableAsStmt *stmt, const char *queryS
 		{
 			TargetEntry *tle = (TargetEntry *)lfirst(elements);
 
-			if (tle->expr && IsA(tle->expr, FuncExpr) && strcasecmp(get_func_name(((FuncExpr *)(tle->expr))->funcid), "identity_into") == 0)
+			if (tle->expr && IsA(tle->expr, FuncExpr) && strncasecmp(get_func_name(((FuncExpr *)(tle->expr))->funcid), "identity_into", strlen( "identity_into")) == 0)
 			{
 				FuncExpr *funcexpr;
 				Oid snamespaceid;
 				char *snamespace;
 				char *sname;
 				List *seqoptions = NIL;
 				ListCell *arg;
-
-				int type_oid;
+				int type_oid = 0;
 				char *type = NULL;
-				TypeName *ofTypename;
-				int64 seed_value;
+				TypeName *type_name = NULL;
+				int64 seed_value = 0;
+				int64 increment_value = 0;
 				int arg_num;
 
 				if (seen_identity)
-				{
 					ereport(ERROR,
-							(errcode(ERRCODE_SYNTAX_ERROR), errmsg("Attempting to add multiple identity columns to table \"%s\" using the SELECT INTO statement.", into->rel->relname)));
-				}
+							(errcode(ERRCODE_SYNTAX_ERROR),
+								errmsg("Attempting to add multiple identity columns to table \"%s\" using the SELECT INTO statement.", into->rel->relname)));
 
 				if (tle->resname == NULL)
-				{
 					ereport(ERROR,
 							(errcode(ERRCODE_SYNTAX_ERROR), errmsg("Incorrect syntax near the keyword 'INTO'")));
-				}
 
 				funcexpr = (FuncExpr *)tle->expr;
 				arg_num = 0;
 				foreach (arg, funcexpr->args)
 				{
-					Node *fargNode = (Node *)lfirst(arg);
-					Const *con;
+					Node *farg_node = (Node *)lfirst(arg);
 					arg_num++;
 					switch (arg_num)
 					{
-						case 1:
```

Comment:
We should throw some error when arg_num reach 4 or more in here ( although it's not possible , but for safe purpose)